pypasta/Repository/Mbox.py

```diff
@@ -454,6 +457,19 @@ def update(self, nofetch):
     def get_lists(self, message_id):
         return self.lists[message_id]
 
+    def latest_message_date(self):
+        latest = datetime.datetime.min
+        for mbox in itertools.chain(self.pub_in, self.mbox_raw.raw_mboxes):
+            f_index = mbox.f_index if isinstance(mbox, PubInbox) \
```

Comment:
Please no isinstance(). Just implement helper functions in both classes that do the job

Reply:
Unfortunately helper functions wouldn't really "help" here. We want the index for a specific mbox. While we can use f_index directly on a pub_in mbox we cannot use something similar on a RawMbox as there is just one RawMbox for multiple lists.